spec.bs

```diff
@@ -859,15 +896,27 @@ To <dfn>fetch trusted signals</dfn> given a [=URL=] |url|, and a [=boolean=] |is
     null ».
   1. If |formatVersion| is not null, is an integer, and is 2:
     1. If |signals|["`keys`"] [=map/exists=], then set |signals| to |signals|["`keys`"].
+    1. TODO: handle priority vector.
     1. Otherwise, return « null, null ».
     1. If |signals| is not an [=ordered map=], return « null, null ».

```

Comment:
this never executes?

Reply:
I'm not sure I understand why? Since above we

```set |signals| to |signals|["`keys`"]```

The new |signals| might possible not be an [=ordered map=], IIUC?

Reply:
my bad. I thought both the if and otherwise block returned.